packages/expo-sqlite/src/hooks.tsx

```diff
@@ -271,7 +271,8 @@ async function openDatabaseWithInitAsync({
   return database;
 }
 
-async function importDatabaseFromAssetAsync(
+// Expose for testing
```

Comment:
After exporting that method we need to add hidden/private annotation if we don't want to expose it in the package docs:
```suggestion
// Expose for testing
/**
 * @private
 */
```